README.md

```diff
@@ -4,7 +4,7 @@
 
 [![Arduino IDE](https://github.com/arduino/arduino-ide/workflows/Arduino%20IDE/badge.svg)](https://github.com/arduino/arduino-ide/actions?query=workflow%3A%22Arduino+IDE%22)
 
-This repository contains the source code of the Arduino IDE 2.x. If you're looking for the old IDE, go to the repository of the 1.x version at https://github.com/arduino/Arduino.
+This repository contains the source code of the Arduino IDE 2.x. If you're looking for the old IDE, go to the (repository of the 1.x version)[https://github.com/arduino/Arduino].
```

Comment:
```suggestion
This repository contains the source code of the Arduino IDE 2.x. If you're looking for the old IDE, go to the [repository of the 1.x version](https://github.com/arduino/Arduino).
```

You reversed the Markdown syntax:

https://www.markdownguide.org/basic-syntax/#links

In the future, please take more care to validate your contributions before submitting a pull request.